### PR TITLE
Add Quadrotor environment with "no_collision" and "velocity_control" two tasks

### PR DESCRIPTION
**Pull request by xueeinstein**

### CODE REVIEW

File: rlschool/quadrotor/env.py [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+U_to_T = np.asarray(
+    [[U_T / m, U_T / m, U_T / m, U_T / m],
+     [-L * U_T / I_1, -L * U_T / I_1, L * U_T / I_1, L * U_T / I_1],
+     [-L * U_T / I_2, L * U_T / I_2, L * U_T / I_2, -L * U_T / I_2],
+     [C * U_T, -C * U_T, C * U_T, -C * U_T]], 'float32')
+T_to_U = np.linalg.inv(U_to_T)
+
+
+class Quadrotor(object):
+    def __init__(self,
```
[the remark this answers is not in the document]
Comment (xueeinstein) [reply rlschool/quadrotor/env.py]: Thanks for your review! Please check the update. Some important notes added.

**Comment by xueeinstein** [comment]
@TomorrowIsAnOtherDay I cannot log in and see the output of TeamCity build. Would you mind pasting the output here?

**Comment by xueeinstein** [comment]
> @TomorrowIsAnOtherDay I cannot log in and see the output of TeamCity build. Would you mind pasting the output here?

TeamCity checks have passed after correctly install the c++ dependencies.

**Comment by TomorrowIsAnOtherDay** [comment]
> @TomorrowIsAnOtherDay I cannot log in and see the output of TeamCity build. Would you mind pasting the output here?
The platform should support anonymous logging, but it does not now. I will update the configuration.

**Pull request merged**